data/axo/Custom.css

```diff
@@ -0,0 +1,175 @@
+/*
+❤️ Support - https://www.patreon.com/axorax
+✨ Change background
+Go to console and type:
+document.querySelector("._11Ivv").style.background = "url('<image-url-here>')"
+*/
+
+._11Ivv {
+    /* Background Start */
+    background: url("https://images.alphacoders.com/596/596630.jpg");
```

Comment:
What's the copyright status of this image? Is it freely available, or do you own it? If not, this could be an issue.

Reply:
my bad ill fix it